### ISSUE TITLE
Error when trying to run iri node

### ISSUE THREAD

**Issue by hotu14** [iotaledger/compass#132]
Hello,
when trying to run the iri node i get the following error

```
user@user-VirtualBox:~/compass/docs/private_tangle$ sudo ./02_run_iri.sh
latest: Pulling from iotaledger/iri
Digest: sha256:77ec44833d0892cb79981e63d212eb743488d365f560e773562f2c5044903297
Status: Image is up to date for iotaledger/iri:latest
WARNING: Published ports are discarded when using host network mode
Logging - property 'logging-level' set to: [INFO]
07/04 13:38:27.945 [main] ERROR IRI$IRILauncher:183 - There was a problem parsing commandline arguments: "--testnet": couldn't convert "--remote" to a boolean
```

**Comment by kwek20** [comment]
Hello @hotu14 
We changed IRI so that all parameters MUST specify true/false. 
Somewhere in there is the parameter "--testnet", without the "true" behind it.

See this PR: https://github.com/iotaledger/iri/pull/1295

**Comment by hotu14** [comment]
thanks. which means i have to add a "true" to all of these parameters?
```
docker run -t --net host --rm -v $scriptdir/db:/iri/data -v $scriptdir/snapsho$
       --testnet="true" \
       --remote \
       --testnet-coordinator $COO_ADDRESS \
       --testnet-coordinator-security-level $security \
       --testnet-coordinator-signature-mode $sigMode \
       --mwm $mwm \
       --milestone-start $milestoneStart \
       --milestone-keys $depth \
       --snapshot /snapshot.txt \
       --max-depth 1000 $@
```

**Comment by GalRogozinski** [comment]
Fixed by #133
